src/Illuminate/View/Compilers/BladeCompiler.php

```diff
@@ -752,4 +759,14 @@ public function withoutComponentTags()
     {
         $this->compilesComponentTags = false;
     }
+
+    /**
+     * Add a handler to be executed before echoing a given class.
+     *
+     * @return void
+     */
+    public function handle(string $className, callable $handler)
```

Comment:
Wondering if this should be `addEchoHandler`. `handle` is so generic and might be confused with a job's `handle` method. The former is a registration of a resolver while the later is an actual resolver.

Reply:
Agreed. I've renamed.